lxd/networks.go

```diff
@@ -23,8 +23,6 @@ import (
 	"github.com/lxc/lxd/lxd/db"
 	"github.com/lxc/lxd/lxd/device"
 	"github.com/lxc/lxd/lxd/dnsmasq"
-	"github.com/lxc/lxd/lxd/instance"
```

Comment:
@keyallis that's where you're removing the import, which you're then re-introducing later. Merging that latter commit with this one will fix things.